ibis-server/tests/routers/ibis/test_bigquery.py

```diff
@@ -220,3 +220,19 @@ def test_query_with_dry_run_and_invalid_sql(self):
         )
         assert response.status_code == 422
         assert response.text is not None
+
+    def test_metadata_list_tables(self):
+        connection_info = self.get_connection_info()
+        response = client.post(
+            url="/v2/ibis/bigquery/metadata/tables",
+            json={"connectionInfo": connection_info},
+        )
+        assert response.status_code == 200
+
+    def test_metadata_list_constraints(self):
+        connection_info = self.get_connection_info()
+        response = client.post(
+            url="/v2/ibis/bigquery/metadata/constraints",
+            json={"connectionInfo": connection_info},
+        )
+        assert response.status_code == 200
```

Comment:
Please assert the contents.

Reply:
I think we do not need to assert the content here, cause Pydentic will raise errors if the responding data structure is incorrect, and we do not care about the actual data in the data structure

Reply:
No, FastAPI only checks the response type is dict. We should check the content like the data count or fields like `name` and `column` should be with it.

Reply:
I'll update the test case and modify the dataset content in another PR